tests/e2e/util.go

```diff
@@ -7670,6 +7670,15 @@ func restartWcpWithWg(ctx context.Context, vcAddress string, wg *sync.WaitGroup)
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
 
+/*
+Stop WCP with WaitGroup
+*/
+func stopWcpWithWg(ctx context.Context, vcAddress string, wg *sync.WaitGroup) {
+	defer wg.Done()
+	err := invokeVCenterServiceControl(ctx, stopOperation, wcpServiceName, vcAddress)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
return proper error, if this util fails due to some reason, let it returns pass/fail error in the parent call and should not fail here